### PR TITLE
No issue: Add unit tests for Context ext function

### PR DESCRIPTION
**Pull request by SubhrajyotiSen**

### CODE REVIEW

File: app/src/test/java/org/mozilla/focus/ext/ContextTest.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        assertFalse(accessibilityManager!!.isEnabled)
+        assertFalse(getAccessibilityManagerInstance().isEnabled)
+    }
+
+    @Throws(Exception::class)
+    private fun getAccessibilityManagerInstance(): AccessibilityManager {
+        return ReflectionHelpers.callStaticMethod(AccessibilityManager::class.java, "getInstance",
+                ReflectionHelpers.ClassParameter.from(Context::class.java, RuntimeEnvironment.application))
+    }
+}
```
Comment (Baron-Severin) [inline app/src/test/java/org/mozilla/focus/ext/ContextTest.kt]: Nitpick: add trailing line break.

File: app/src/test/java/org/mozilla/focus/ext/ContextTest.kt [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    fun startUp() {
+        context = RuntimeEnvironment.application
+        accessibilityManager = context!!.getAccessibilityManager()
+    }
+
+    @Test
+    fun `isScreenReaderEnabled() returns true when TouchExploration enabled`() {
+        shadowOf(accessibilityManager).isTouchExplorationEnabled = true
+        assertTrue(context!!.isVoiceViewEnabled())
+    }
```
Comment (Baron-Severin) [inline app/src/test/java/org/mozilla/focus/ext/ContextTest.kt]: Nitpick: add line break between methods.

File: app/src/test/java/org/mozilla/focus/ext/ContextTest.kt [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    private var accessibilityManager: AccessibilityManager? = null
+
+    @Before
+    fun startUp() {
+        context = RuntimeEnvironment.application
+        accessibilityManager = context!!.getAccessibilityManager()
+    }
+
+    @Test
+    fun `isScreenReaderEnabled() returns true when TouchExploration enabled`() {
```
Comment (Baron-Severin) [inline app/src/test/java/org/mozilla/focus/ext/ContextTest.kt]: Nitpick: Shouldn't this method be called `isVoiceViewEnabled() returns true when TouchExploration enabled`?

File: app/src/test/java/org/mozilla/focus/ext/ContextTest.kt [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        shadowOf(accessibilityManager).isTouchExplorationEnabled = true
+        assertTrue(context!!.isVoiceViewEnabled())
+    }
+    @Test
+    fun `isScreenReaderEnabled() returns false when TouchExploration disabled`() {
```
Comment (Baron-Severin) [inline app/src/test/java/org/mozilla/focus/ext/ContextTest.kt]: Nitpick: suggest method name change, same as above.

File: app/src/test/java/org/mozilla/focus/ext/ContextTest.kt (hunk 3 above)
Comment (SubhrajyotiSen) [reply app/src/test/java/org/mozilla/focus/ext/ContextTest.kt]: Yes. It should be. It was picked from a commit from android-components and I forgot to update method names accordingly

**Pull request merged**